docs/sprint-planning.md

```diff
@@ -0,0 +1,24 @@
+---
+id: sprint-planning
+title: Sprint Planning
+slug: /sprint-planning
+---
+
+1. Go to the "Stories" page in Shortcut
+1. Go to the "Needs Iteration" Space
+1. Assign the current Iteration to any Story that does not have an Interation and is in progress
+1. Go to the "Needs Points" Space
+1. Assign Points to any Stories that are unestimated
+1. Go to the "Everything" Space
+1. Uncheck the "No Iteration" Filter
+1. Go through each Column starting with "Acceptance", discuss the status of each Story, and take action if needed
+1. If any Story in the "Ready For Development" Column will not be started during the current Iteration move them to the next Iteration
+1. Go to the "Reports" page
+1. Look at the last 3 months to determine the team's average velocity
+1. Go back to the "Everything" Space on the "Stories" page
+1. Review and assign WordPress Legacy work that needs to be prioritized for the next Iteration
+1. Go to the "Milestone" page for each team
```

Comment:
by "each team" do you mean Software & Product?

Reply:
No, I mean each software team. Right now "Product Activation" and "Marketplace".

Reply:
ah ok, then maybe something like
> Go to the page for each Milestone that's in progress

?